packages/block-editor/src/components/block-inspector/style.scss

```diff
@@ -41,6 +41,7 @@
 }
 
 .block-editor-block-inspector__tab-item {
+	bottom: -1px;
```

Comment:
Would it make sense to use -$border-width instead of the hardcoded -1px? ($border-width=1px).